src/components/Modal/BaseModal.js

```diff
@@ -110,8 +110,8 @@ class BaseModal extends PureComponent {
                 animationOut={this.props.animationOut || animationOut}
                 useNativeDriver={this.props.useNativeDriver}
                 hideModalContentWhileAnimating={this.props.hideModalContentWhileAnimating}
-                animationInTiming={this.props.animationInTiming}
-                animationOutTiming={this.props.animationOutTiming}
+                animationInTiming={this.props.disableAnimation ? 1 : this.props.animationInTiming}
+                animationOutTiming={this.props.disableAnimation ? 1 : this.props.animationOutTiming}
```

Comment:
Not happy with this change. Can't we just follow the same pattern as we do on the web?

Reply:
Updated the pull request with a recommended change. 😊